src/PrestaShopBundle/Controller/Admin/Configure/AdvancedParameters/FeatureFlagController.php

```diff
@@ -0,0 +1,76 @@
+<?php
+/**
+ * Copyright since 2007 PrestaShop SA and Contributors
+ * PrestaShop is an International Registered Trademark & Property of PrestaShop SA
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.md.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://devdocs.prestashop.com/ for more information.
+ *
+ * @author    PrestaShop SA and Contributors <contact@prestashop.com>
+ * @copyright Since 2007 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ */
+
+declare(strict_types=1);
+
+namespace PrestaShopBundle\Controller\Admin\Configure\AdvancedParameters;
+
+use PrestaShopBundle\Controller\Admin\FrameworkBundleAdminController;
+use PrestaShopBundle\Security\Annotation\AdminSecurity;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+
+/**
+ * Manages the "Configure > Advanced Parameters > Experimental Features" page.
+ */
+class FeatureFlagController extends FrameworkBundleAdminController
+{
+    public const CONTROLLER_NAME = 'AdminAdvancedParametersFeatureFlags';
```

Comment:
Seems like a useless const?

Reply:
I added it "just in case" because last time I tried to remove something "I dont why it's needed", I totally broke the authorization on my BO page 😅  .

I plan to write a documentation after feature freeze about "what do you need to create a new BO page (NOT migrate an existing legacy BO page)" and I will research what each needed item does exactly so I will find the answer eventually.

Reply:
AFAIK it's only needed if you do something like this: https://github.com/PrestaShop/PrestaShop/blob/develop/src/PrestaShopBundle/Controller/Admin/Configure/AdvancedParameters/MemcacheServerController.php#L88

Reply:
Fixed in https://github.com/PrestaShop/PrestaShop/pull/23703/commits/bb2f40a2a10a847f5f1144549532bff524cc3074